POLICY.md

```diff
@@ -0,0 +1,40 @@
+# Policy
+
+This document will lay out the official policies used within Pulsar-Edit. This document lists the various policies that have come about naturally as the project progressed and so deserve to be properly documented and viewable by anyone.
+
+## Polls
+
+A cornerstone of the way the Pulsar team interacts with the community is via Polls. These polls are usually created on Discord, but may in the future be expanded to other platforms as needed.
+
+A Poll is used to allow the community at large to easily and quickly contribute to a conversation and provide their opinions on future development or goals of Pulsar-Edit.
+
+* Whenever we need to make a decision that will affect a large percentage of users/contributors, make a substantial enough change, drastically affect a small percentage of users/contributors then a poll must be made.
+* A Poll must be available for voting for at least 24 hours to accommodate different timezones and work schedules. Polls may be longer than this, but must never be shorter.
+* If a poll fails, then you must not recreate the exact same poll right afterwards. If the same idea has new merit or new validation at a later time, communicate with the community to determine if another poll is reasonable.
```

Comment:
LONG VERSION

<details><summary>(personal philosophical rambling, skip to short version if need be, click to expand):</summary>

I know we _do_ polls.

I dunno about the "must" wording here, since we are not utterly consistent about it, and sometimes a PR needs to go out, say for security reasons for example, or there _is no consensus_ and yet somebody needs to pick something or move something forward.

Like, polls are IMO more for when we're not already sure as an admin team. We don't ask for input when there's already an obvious answer, even when that obvious answer affects a lot of people or some important constituency group.

So I think in all honestly the polls have been, and should be, a function of when the admins want/need help making the right call. If we require it based on who or how many people it affects, that may not align with what is practical and what in all honestly drives current poll behavior (in my personal view and observations).

Also: When this says "must" -- who is enforcing this, and what is the enforcement mechanism? If it's not a "must", or if it's not enforced, then this strikes me as more of a guideline, and like with IETF RFC's, I think these should be "should" rather than "must", and clarify that this is a guideline and documentation of an ongoing practice, moreso than a rule.

While community is a major, major aligning principle of the project, community members who aren't mods are neither as engaged in every little decision we might have to make, nor IMO could they or should they be ultimately responsible over everything, there has to be an admin team for things to land on good and bad, and this starts to feel overly prescriptive as written in my personal view. Though we do obviously take community perspective on things pretty often. But it's often in terms of who pops into issues and who is in the Discord server, not _necessarily_ a poll. Heck, I hardly consider the admin team much distinct from the rest of the community other than I know they will show up, and they have the responsibility associated with the level of access they are granted. Other than that we _are_ community members. And the informal nature of our association is shown by how many decisions we tend to make without polls, there is, thus far, a trust that is shared, IMO, to move in the right direction and everybody pitch in as and when needed.

I think implicitly a lot of the "community" vibe of Pulsar is that there is an _existing_ community of Atom users who would be sad to see an editor like Atom go without a viable similar replacement, and that by respecting the wish of many people to have a viable and similar replacement, we are reaching to meet the (often unspoken or informally organized) demand for such things to be available.

So community is a principle, but I don't think you can prescribe respect for community as a rule, it's ethos and it manifests in how we do everything, but it still is a project that needs a leadership team and executive decisions to be made, etc. So we can't mandate polls and have it be smooth and well functioning, or at least I am skeptical of the implication that we _already_ do that and skeptical of the requirement that we should _start_ doing that.

</details>

---

SHORT VERSION:

Okay, so, getting off my soap box.

I would suggest changing "must" in this passage to "should", or even consider just describing this as "a thing we do" and not prescribe it or suggest it is necessary in any way. Just that it's a convention and we prefer it in a lot of situations where we need community input.

Like if we drift to not ever doing polls, something would have gone "off"/wrong compared to how this document describes things.

Reply:
Wow this is very very well written. And honestly I totally respect what you are saying here, and side with you in agreeance.

Guess my brain has left out all the larger decisions we made because it was obvious, or because it was the only right one. Forgetting that these guidelines went against that.

I'll go ahead and add this in, and likely may include some of what you've said here as well.

Appreciate your input, and seeing this from another point of view!

Reply:
@DeeDeeG changes have been made as requested

Reply:
Are the commits pushed to the PR branch? I can't find it on here.

Reply:
@DeeDeeG Sorry, they were in my local stash my bad